sympy/solvers/solvers.py

```diff
@@ -2529,15 +2569,22 @@ def solve_linear_system_LU(matrix, syms):
 
 
 def det_perm(M):
-    """Return the det(``M``) by using permutations to select factors.
-    For size larger than 8 the number of permutations becomes prohibitively
+    """
+    Return the det *M* by using permutations to select factors.
```

Comment:
The parentheses here were part of the mathematical notation. This can be rewritten as `the determinant of *M*`.